my_first_bioinf_package.py

```diff
@@ -0,0 +1,69 @@
+from inner_code import filter_gc_content, filter_length, filter_quality
+from typing import Tuple, Union
+def filter_fastq (seqs: dict, length_bounds: Union [tuple, int] =(0, 2**32), gc_bounds: Union [tuple, int] =(0, 100), quality_threshold: int=0) -> dict: 
+    """  
+    Returns filtered sequences\n 
+    Arguments:\n
+    seqs (dict) - the dictionary with the sequences that should be filtered;\n
+    gc_bounds (tuple or int) - the boundaries for gc-content; default meaning - (0; 100);\n  
+    length_bounds (tuple or int) - the boundaries for length; default meaning - (0, 2**32);\n
+    quality_threshold (int) - the number for quality threshold; default meaning - 0\n
+    Return:\n
+    dict - filtered sequences, collected in a dictionary  

```

Comment:
Комментарии в докстринге не требуют разделителя строк \n, он так и выводится в тексте, что не очень удобно
```suggestion
    Returns filtered sequences
    Arguments:
    seqs (dict) - the dictionary with the sequences that should be filtered;
    gc_bounds (tuple or int) - the boundaries for gc-content; default meaning - (0; 100);
    length_bounds (tuple or int) - the boundaries for length; default meaning - (0, 2**32);
    quality_threshold (int) - the number for quality threshold; default meaning - 0
    Return:
    dict - filtered sequences, collected in a dictionary  

```